e2e_tests/src/e2e_conduit/scenarios/__init__.py

```diff
@@ -0,0 +1,22 @@
+from e2e_conduit.fixtures.plugin_fixture import PluginFixture
+
+
+class Scenario:
```

Comment:
Should this class go into its own file `Scenario.py`? It's really a stylistic choice. I find it easier to know which classes are available by scanning a module directory's contents and often overlook `__init__.py`, but in some org's `__init__.py` is known to be important to always look at.